### ISSUE TITLE
Staging Review finding: Odd screen reader behavior for refill success alert

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#84223]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** MHV-Medications
**Product:** MHV Medications
**Feature:** Medications

## Findings details
**VA.gov Experience Standard - issue:** [User encounters design components that are inconsistent or confusing.](https://design.va.gov/about/experience-standards/#user-encounters-consistent-clear-design-components)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On [/medications/refill](https://staging.va.gov/my-health/medications/refill) when successfully refilling a prescription, I'm noticing some odd behavior in how screen readers announce the success alert. Parts of the text of the alert are announced multiple times, especially on NVDA.

Keyboard focus is set to the alert when the refill is requested, which should ensure that it's the first thing announced by screen readers. But the `va-alert` component also has `role="alert"` and `aria-live="polite"` attributes on it, which also instruct screen readers to announce the contents of the alert. I think the duplication may be causing issues.

## Recommended action

I think you're safe to remove `role="alert"` and `aria-live="polite"` and just rely on keyboard focus. Those attributes aren't baked into the success variation of `va-alert` (reserved for the error alert as the most-critical alert type), and I don't think they buy us anything here that's not already accomplished by focus management.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** [Alert ](https://design.va.gov/components/alert)
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by IdeaTrail** [comment]
Completed with Jira item: https://jira.devops.va.gov/browse/MHV-58722
